frontend/components/FormComponents.tsx

```diff
@@ -0,0 +1,416 @@
+import 'react-datepicker/dist/react-datepicker.css'
+
+import { ContentState, convertToRaw, EditorState } from 'draft-js'
+import draftToHtml from 'draftjs-to-html'
+import { useFormikContext } from 'formik'
+import Head from 'next/head'
+import React, {
+  ReactElement,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import DatePicker from 'react-datepicker'
+import Select from 'react-select'
+import s from 'styled-components'
```

Comment:
```suggestion
import styled from 'styled-components'
```
I know this hasn't been done consistently in the codebase, but I really think we should move towards `styled` rather than `s` -- it's recommended in the docs and also prettier will pick up CSS and lint it if it's inside a `styled` block but it doesn't know what `s` is!

Reply:
We currently have the entire codebase as `s` with no exceptions, so I think for this PR it would be nice to keep it that way.

However, I'm down to do a mass migration from `s` to `styled` at some point, should just be a massive find and replace.

Reply:
Created a task in [ch2813].